account_financial_report/wizard/aged_partner_balance_wizard.py

```diff
@@ -27,13 +27,54 @@ class AgedPartnerBalanceWizard(models.TransientModel):
         default="posted",
     )
     account_ids = fields.Many2many(
-        comodel_name="account.account", string="Filter accounts"
+        comodel_name="account.account",
+        string="Filter accounts",
+        domain=[("reconcile", "=", True)],
+        required=True,
     )
     receivable_accounts_only = fields.Boolean()
     payable_accounts_only = fields.Boolean()
     partner_ids = fields.Many2many(comodel_name="res.partner", string="Filter partners")
     show_move_line_details = fields.Boolean()
 
+    account_code_from = fields.Many2one(
```

Comment:
Isn't possible to put these fields and onchange in `account_financial_report_abstract_wizard` so you don't repeat so much code?

Reply:
What about this question?

Reply:
Yes, sorry.

The on_change_account_range method is different for open items and aged balance comparing to the general ledger and trial balance report.

I think this will not save on a lot of code.

Reply:
Why are they different? At the end, they should fill the accounts in a m2m field.

Reply:
For Open Items and Aged partner balance, the domain of the account_code_from and account_code_to is only for accounts that are reconcilable and the selected accounts are only the reconciliable ones between this two codes too.

Reply:
Well, then you can filter them through a hook. This refactoring is supposed to reduce the spaghetti code for doing the module more maintainable, so let's do it that way for all!